db/pf-schema-X.Y.Z.sql

```diff
@@ -1107,6 +1107,78 @@ CREATE TABLE radius_audit_log (
 ) ENGINE=InnoDB;
 
 --
+-- Table structure for table `node_option82`
+--
+
+CREATE TABLE `node_option82` (
+  `node_option82_id` int NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  `created_at` TIMESTAMP NOT NULL,
+  `mac` varchar(17) NOT NULL,
+  `option82_switch` varchar(17) NULL,
+  `switch_id` varchar(17) NULL,
+  `port` varchar(8) NOT NULL default '',
+  `vlan` varchar(255) default NULL,
+  `circuit_id_string` varchar(255) default NULL,
+  `module` varchar(255) default NULL,
+  `host` varchar(255) default NULL,
+  UNIQUE KEY mac (mac)
+) ENGINE=InnoDB;
+
+--
+-- Table structure for table `node_option82_history`
+--
+
+CREATE TABLE `node_option82_history` (
+  `node_option82_id` int NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  `created_at` TIMESTAMP NOT NULL,
+  `mac` varchar(17) NOT NULL,
+  `option82_switch` varchar(17) NULL,
+  `switch_id` varchar(17) NULL,
+  `port` varchar(8) NOT NULL default '',
+  `vlan` varchar(255) default NULL,
+  `circuit_id_string` varchar(255) default NULL,
+  `module` varchar(255) default NULL,
+  `host` varchar(255) default NULL,
+  INDEX (mac)
+) ENGINE=InnoDB;
+
+--
+-- Trigger to archive node_option82 entries to the history table after an update
+--
+
+DROP TRIGGER IF EXISTS node_option82_after_update_trigger;
+DELIMITER /
+CREATE TRIGGER node_option82_after_update_trigger AFTER UPDATE ON node_option82
+FOR EACH ROW
+BEGIN
+    INSERT INTO node_option82_history
```

Comment:
Why we are not keeping the 'node_option82_id' ?
Wouldn't that create some "duplicates" if consulting both the "active" (node_option82) and "history" (node_option82_history) tables ?